python_modules/dagster-graphql/dagster_graphql/schema/partition_sets.py

```diff
@@ -358,7 +358,7 @@ def _get_time_partitions_metadata(partition_def_data):
 
             partitions_def = partition_def_data.get_partitions_definition()
             return GrapheneTimePartitionsDefinitionMetadata(
-                startTime=partitions_def.start_time,
+                startTime=partitions_def.start,
```

Comment:
hey @clairelin135  I had to change this to get Dagit to load the asset graph for me - does this make sense?

Reply:
Hi @bengotow ah that's my mistake! I think we actually want `partitions_def.start.timestamp()` as that will generate the float value instead of returning the datetime object.

But I've updated my PR to fix this so you should be able to rebase and get this new fix